templates/advantage/index.html

```diff
@@ -7,18 +7,19 @@
 {% block content %}
 
 {% if openid %}
-<section class="p-strip--suru-topped">
+<section class="p-strip--suru-topped is-shallow" style="padding-top: 6rem">
   <div class="row">
-    <div class="col-12">
-      <h1 class="u-no-max-width">Ubuntu Advantage for Infrastructure</h1>
+    <div class="col-6">
+      <h1>Ubuntu Advantage for Infrastructure</h1>
+      <p>By using Ubuntu Avantage you agree to our <a href="/legal/ubuntu-advantage-service-description">terms and conditions</a>.</p>
```

Comment:
This statement is necessary, but makes sense mainly as something you review just before you get your free token. Which tells me:
- we still need a one-off button for that specific purpose (“Get my free token”)
- this statement should be above it.